sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ThroughputProperties.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Offer;
+import com.azure.cosmos.implementation.OfferAutoscaleSettings;
+
+/**
+ * Represents throughput of the resources in the Azure Cosmos DB service.
+ */
+public class ThroughputProperties {
+    private final Offer offer;
+
+    ThroughputProperties(Offer offer) {
```

Comment:
How do CX UT?

Reply:
Will try to redesign to make offer mockable in subsequent PRs.